src/logs.ts

```diff
@@ -1,4 +1,13 @@
 export const enum LogLevel {
+  Trace = 'trace',
```

Comment:
Personal Note: `LogLevel` is a type that has been defined by other 3000 libraries out there. Does it make sense to define our own levels as well (which are exactly equal to the one that everybody else is offering?) Maybe we can evaluate the idea of integrating with somebody else? Food for thoughts.

Reply:
Yes, see line `19-20` in that file :). For now, we just need the levels and not an actual logging system. Purposefully designed the levels to be in line with what other libraries have done so that we can easily add a logging library when we need a formal logger.